components/list/list-item-drag-mixin.js

```diff
@@ -0,0 +1,120 @@
+import { getUniqueId } from '../../helpers/uniqueId.js';
+import { html } from 'lit-element/lit-element.js';
+import { nothing } from 'lit-html';
+
+export const ListItemDragMixin = superclass => class extends superclass {
+
+	static get properties() {
+		return {
+			draggable: {type: Boolean, reflect: true },
+			key: { type: String, reflect: true }
+		};
+	}
+
+	constructor() {
+		super();
+		this._itemDragId = getUniqueId();
+	}
+
+	connectedCallback() {
+		super.connectedCallback();
+		if (this.key === undefined) {
+			this.draggable === undefined;
+		}
+	}
+
+	_dispatchDragEvent(detail) {
+		this.dispatchEvent(new CustomEvent('d2l-list-item-position', {
+			detail,
+			bubbles: true
+		}));
+	}
+
+	keyboardMode(isEnabled) {
+		if (isEnabled) {
+			this.dispatchEvent(new CustomEvent('d2l-list-item-drag-keyboard-mode', {
+				detail: null,
+				bubbles: true
+			}));
+		}
+	}
+
+	_renderDragHandle() {
+		return this.draggable ? html`
+			<div
+				id="${this._itemDragId}"
+				class="d2l-list-item-drag"
+				.draggable="${this.draggable}"
```

Comment:
Are we using this attribute anywhere on the drag handle itself?

Reply:
Nope, I don't think so.

This should be `?draggable="${this.draggable}"` I feel like Zina's point is this doesn't even need to be here though.